InitNiKom/InitNiKom.c

```diff
@@ -28,7 +28,7 @@ void createDirOrDie(char *dirname) {
   BPTR lock;
   struct FileInfoBlock __aligned fib;
   
-  if(lock = Lock(dirname, ACCESS_READ)) {
+  if((lock = Lock(dirname, ACCESS_READ))) {
```

Comment:
Why does gcc want these extra parens..? It's not like there is any ambiguity without them.